Camera lock fix for Android 7.1.1 AOSP

pschucker opened the pull request:
Releases camera locks on Android 7.1.1 AOSP. Tested with Google Nexus 5. Restarting cameraserver did not solve the problem but restarting mm-qcamera-daemon finally released the camera.

androidseb commented:
Thank you for the suggestion :-)
The code you proposed did not compile properly and the text was overlaying the icon, so I fixed it by adding the text below.
I also changed the imageview to be a button, that way the click feedback is visible.
Here is the commit link:
[https://github.com/androidseb/camerarestarter/commit/3b2abd57c319e89ffb486e717c4cb6d2f5da4313](https://github.com/androidseb/camerarestarter/commit/3b2abd57c319e89ffb486e717c4cb6d2f5da4313)

Saeed-Pooyanfar commented:
Thanks for update ...
But widget became small now!
I just forgot to add RelativeLayout properties!
My code should works fine !
On Apr 3, 2016 10:25 PM, "androidseb" <notifications@github.com> wrote:

> Closed #1 <https://github.com/androidseb/camerarestarter/pull/1>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/androidseb/camerarestarter/pull/1#event-612795120>
>

androidseb commented:
I tested it the widget size was fine... Maybe I missed something, can you send me a screenshot of the rendering you have please?

Saeed-Pooyanfar commented:
Yes, here you are ...
I think it's small compare with pervious version ...
On Apr 4, 2016 4:26 AM, "androidseb" <notifications@github.com> wrote:

> I tested it the widget size was fine... Maybe I missed something, can you
> send me a screenshot of the rendering you have please?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/androidseb/camerarestarter/pull/1#issuecomment-205079504>
>

pschucker committed:
Camera lock fix for Android 7.1.1 AOSP

Releases camera locks on Android 7.1.1 AOSP. Tested with Google Nexus 5. Restarting cameraserver did not solve the problem but restarting mm-qcamera-daemon released the camera.
